web/src/styles.css

```
@@ -10,28 +10,28 @@
+  --muted: #f0f0f2;
   --muted-foreground: #6b6b74;
   --accent: #e7e8fa;
   --accent-foreground: #343d9a;
   --destructive: #bf3b49;
-  --border: #dedee4;
-  --input: #d7d7df;
+  --border: #e3e3e7;
+  --input: #ddddE3;
   --ring: #6974d8;
   --radius: 0.75rem;
   --serif: "Castoro", "Iowan Old Style", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)